### ISSUE TITLE
Questions of Inconsistency Between the Paper and the Code Regarding Training Settings

### ISSUE THREAD

**Issue by LongfeiH** [SYZhang0805/DisBack#5]
First of all, I would like to express my respect for your work, but I have some questions regarding the inconsistency between the paper and the code in terms of training settings. Specifically:

In Appendix B.2 of the paper, it is stated that the pre-trained weights are from SDXL, but in the code, the pre-trained weights are from DMD2. In addition, the learning rate, training steps, etc., are also different. I would also like to ask: when replacing trajectory points as the training target, should the entire guidance model's weights be replaced, or only the real UNet?

Due to these concerns, I have been unable to successfully reproduce your work. I hope you can provide some clarification.

**Comment by yuanzhi-zhu** [comment]
I also found this inconsistency very discouraging, and I am wondering about the setup of ImageNet experiments, too.

@LongfeiH Could you share some results of your expr, would be very helpful.

**Comment by yuanzhi-zhu** [comment]
Moreover, it seems diffusion GAN is used in the training script?
https://github.com/SYZhang0805/DisBack/blob/d015b5e5c552666881a967cbee245ca5e895190b/experiments/sdxl/sdxl_cond399_8node_lr5e-7_1step_diffusion1000_gan5e-3_guidance8_noinit_noode.sh#L42-L43

**Comment by SYZhang0805** [comment]
> First of all, I would like to express my respect for your work, but I have some questions regarding the inconsistency between the paper and the code in terms of training settings. Specifically:首先，我想表达我对你的工作的尊重，但我对论文和代码在训练设置方面的不一致有一些疑问。具体说来：
> 
> In Appendix B.2 of the paper, it is stated that the pre-trained weights are from SDXL, but in the code, the pre-trained weights are from DMD2. In addition, the learning rate, training steps, etc., are also different. I would also like to ask: when replacing trajectory points as the training target, should the entire guidance model's weights be replaced, or only the real UNet?在论文的附录 B.2 中，指出预训练的权重来自 SDXL，但在代码中，预训练的权重来自 DMD2。此外，学习速度、训练步骤等也不同。我还想问：在替换轨迹点作为训练目标时，应该替换整个引导模型的权重，还是只替换真实的 UNet？
> 
> Due to these concerns, I have been unable to successfully reproduce your work. I hope you can provide some clarification.由于这些担忧，我一直无法成功复制您的作品。我希望你能提供一些澄清。

Sorry for replying late, the pre-trained weights are from SDXL, the learning rate and training steps should be the same as the paper. When replacing trajectory points as the training target, the entire guidance model's weights are replaced by the checkpoint in the degradation path.

**Comment by SYZhang0805** [comment]
> I also found this inconsistency very discouraging, and I am wondering about the setup of ImageNet experiments, too.我还发现这种不一致非常令人沮丧，我也想知道 ImageNet 实验的设置。
> 
> [@LongfeiH](https://github.com/LongfeiH) Could you share some results of your expr, would be very helpful.您能否分享一些 expr 的结果，将非常有帮助。

The ImageNet experiment is modified based on the Diff-Instruct code. You can implement it according to the experimental design described in the paper.
